cms/djangoapps/contentstore/management/commands/tests/test_fix_not_found.py

```diff
@@ -0,0 +1,64 @@
+"""
+Tests for the fix_not_found management command
+"""
+
+from django.core.management import call_command
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from xmodule.modulestore.split_mongo import BlockKey
+
+
+class TestFixNotFound(ModuleStoreTestCase):
+    """
+    Tests for the fix_not_found management command
+    """
+    def test_fix_not_found_non_split(self):
+        course = CourseFactory(default_store=ModuleStoreEnum.Type.mongo)
+        with self.assertRaises(SystemExit):
+            call_command("fix_not_found", unicode(course.id))
+
+    def test_fix_not_found(self):
+        course = CourseFactory(default_store=ModuleStoreEnum.Type.split)
+        # pylint: disable=protected-access
+        store = self.store._get_modulestore_for_courselike(course.id)
+        course_locator = course.id.for_branch(ModuleStoreEnum.BranchName.draft)
+        ItemFactory.create(category='sequential', parent_location=course.location)
+
+        # create a dangling usage key that we'll add to the course structure
+        dangling_pointer = course_locator.make_usage_key('sequential', 'ImNotThere')
+
+        # pylint: disable=protected-access
+        original_structure = store._lookup_course(course_locator).structure
+        index_entry = store._get_index_if_valid(course_locator)
+        new_structure = store.version_structure(
+            course_locator, original_structure, ModuleStoreEnum.UserID.mgmt_command
+        )
+
+        # add the empty pointer as a child of `course`
+        course_block_key = BlockKey.from_usage_key(course.location)
+        course_children = new_structure['blocks'][course_block_key].fields['children']
+
+        # update the course_structure
+        new_structure['blocks'][course_block_key].fields['children'].append(
+            BlockKey.from_usage_key(dangling_pointer)
+        )
+        store.update_structure(course_locator, new_structure)
+        # pylint: disable=protected-access
+        store._update_head(course_locator, index_entry, course_locator.branch, new_structure['_id'])
```

Comment:
@cpennington , probably don't need the low level manipulation—I just didn't think to use update_item. I'll try it out without that.

BTW, I closed this PR because I'm opening a new PR with all my recent orphan fixes to go against the RC. It's easier to do it as one PR, since so many of the changes are so intertwined